src/app/onion/learning-object-builder/components/builder-navbar/builder-navbar.component.html

```diff
@@ -52,7 +52,7 @@
         <button
           id="submit"
           class="button"
-          *ngIf="['unreleased', 'rejected'].includes(learningObject?.status)"
+          *ngIf="['unreleased'].includes(learningObject?.status)"
```

Comment:
what does this line do?

Reply:
This button is for submitting for review.  So what this line is doing is checking if the learning object's status is unreleased because only unreleased objects can be submitted for review.